tests/nosetests/pyanaconda_tests/module_storage_test.py

```diff
@@ -1203,6 +1405,150 @@ def format_test(self, blockdev):
         ])
 
 
+class ISCSIInterfaceTestCase(unittest.TestCase):
+    """Test DBus interface of the iSCSI module."""
+
+    def setUp(self):
+        """Set up the module."""
+        self.iscsi_module = ISCSIModule()
+        self.iscsi_interface = ISCSIInterface(self.iscsi_module)
+
+        self._portal = Portal()
+        self._portal.ip_address = "10.43.136.67"
+        self._portal.port = "3260"
+
+        self._credentials = Credentials()
+        self._credentials.username = "mersault"
+        self._credentials.password = "nothing"
+        self._credentials.reverse_username = "tluasrem"
+        self._credentials.reverse_password = "thing"
+
+        self._node = Node()
+        self._node.name = "iqn.2014-08.com.example:t1"
+        self._node.address = "10.43.136.67"
+        self._node.port = "3260"
+        self._node.iface = "iface0"
+        self._node.net_ifacename = "ens3"
+
+        # Connect to the properties changed signal.
+        self.callback = Mock()
+        self.iscsi_interface.PropertiesChanged.connect(self.callback)
+
+    @patch('pyanaconda.modules.storage.iscsi.iscsi.iscsi')
+    def initator_property_test(self, iscsi):
+        """Test Initiator property."""
+        initiator_name = "iqn.1994-05.com.redhat:blabla"
+        iscsi.initiator_set = False
+        self.iscsi_interface.SetInitiator(initiator_name)
+        iscsi.initiator = initiator_name
+        self.assertEqual(self.iscsi_interface.Initiator, initiator_name)
+        iscsi.initiator_set = True
+        initiator_name2 = "iqn.1994-05.com.redhat:blablabla"
+        self.iscsi_interface.SetInitiator(initiator_name2)
+        self.callback.assert_called_once_with(ISCSI.interface_name, {'Initiator': initiator_name}, [])
+
+    @patch('pyanaconda.modules.storage.iscsi.iscsi.iscsi')
+    def can_set_initiator_test(self, iscsi):
+        """Test CanSetInitiator method."""
+        self.iscsi_interface.CanSetInitiator()
```

Comment:
You don't test the returned value.

Reply:
I didn't want to mock blivet's iscsi to a specific value in this case just to check its negation (implementation just returns "not iscsi.initiator_set") to pretend that I am actually testing something here, I just wanted to check that the method is callable. Do you think mocking iscsi.initiator_set would make sense here?

Reply:
Ok, but you can at least test that it returns a boolean value :-)

Reply:
Done